code/datums/votes/_vote_datum.dm

```diff
@@ -161,17 +173,48 @@
  * Return a formatted string of text to be displayed to everyone.
  */
 /datum/vote/proc/get_result_text(list/all_winners, real_winner, list/non_voters)
-	if(length(all_winners) <= 0 || !real_winner)
-		return span_bold("Vote Result: Inconclusive - No Votes!")
-
 	var/returned_text = ""
 	if(override_question)
 		returned_text += span_bold(override_question)
 	else
 		returned_text += span_bold("[capitalize(name)] Vote")
 
+	returned_text += "\nWinner Selection: "
+	switch(winner_method)
+		if(VOTE_WINNER_METHOD_NONE)
+			returned_text += "None"
+
+		if(VOTE_WINNER_METHOD_RANDOM)
+			returned_text += "Random"
+
+		else
+			returned_text += "Simple"
+
+	var/total_votes = 0 // for determining percentage of votes
+	for(var/option in choices)
+		total_votes += choices[option]
+
+	if(total_votes <= 0)
+		return span_bold("Vote Result: Inconclusive - No Votes!")
+
+	returned_text += "\nResults:"
 	for(var/option in choices)
-		returned_text += "\n[span_bold(option)]: [choices[option]]"
+		returned_text += "\n"
+		var/votes = choices[option]
+		var/percentage_text = ""
+		if(votes > 0)
```

Comment:
Is this necessary? `0 / ` etc is just zero. Seems like a waste of code

Reply:
log of 0 is a runtime error